vulkano/src/device/physical.rs

```
@@ -929,3 +1087,240 @@ impl From<ash::vk::ShaderCorePropertiesFlagsAMD> for ShaderCoreProperties {
+    /// image view.
+    pub storage_write_without_format: bool,
+    /// Can be used with a color attachment in a framebuffer, or with an input attachment
+    /// descriptor.
+    pub color_attachment: bool,
+    /// Can be used with a color attachment in a framebuffer with blending, or with an input
+    /// attachment descriptor.
+    pub color_attachment_blend: bool,
+    /// Can be used with a depth/stencil attachment in a framebuffer, or with an input attachment
+    /// descriptor.
+    pub depth_stencil_attachment: bool,
+    /// Can be used with a fragment density map attachment in a framebuffer.
+    pub fragment_density_map: bool,
+    /// Can be used with a fragment shading rate attachment in a framebuffer.
+    pub fragment_shading_rate_attachment: bool,
+    /// Can be used with the source image in a transfer (copy) operation.
+    pub transfer_src: bool,
+    /// Can be used with the destination image in a transfer (copy) operation.
+    pub transfer_dst: bool,
+    /// Can be used with the source image in a blit operation.
+    pub blit_src: bool,
+    /// Can be used with the destination image in a blit operation.
+    pub blit_dst: bool,
+
+    // Sampling
+    /// Can be used with samplers or as a blit source, using the
+    /// [`Linear`](crate::sampler::Filter::Linear) filter.
+    pub sampled_image_filter_linear: bool,
+    /// Can be used with samplers or as a blit source, using the
+    /// [`Cubic`](crate::sampler::Filter::Cubic) filter.
+    pub sampled_image_filter_cubic: bool,
+    /// Can be used with samplers using a reduction mode of
+    /// [`Min`](crate::sampler::SamplerReductionMode::Min) or
+    /// [`Max`](crate::sampler::SamplerReductionMode::Max).
+    pub sampled_image_filter_minmax: bool,
+    /// Can be used with sampler YCbCr conversions using a chroma offset of
+    /// [`Midpoint`](crate::sampler::ycbcr::ChromaLocation::Midpoint).
```

@Rua Cargodoc complains about `ycbcr` submodule. Other than that I didn't find any issues.

---

I have a PR in the works that adds that submodule, but I put it on hold to work on other things first. You can ignore this error.

---

Ok!